dbotconf/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	github.com/spf13/cobra v1.3.0
 	github.com/stretchr/testify v1.7.0
-	go.opentelemetry.io/build-tools v0.0.0-20220214221246-ac3bcee21caa
+	go.opentelemetry.io/build-tools v0.0.0-00010101000000-000000000000
```

Comment:
Shouldn't this depend on an actual (pseudo)version of this module?  The commit where `FindMods` was introduced should now be known and referenceable.

Reply:
Updated.